### DISCUSSION TITLE
Green Software Design resources

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/patterns#389] [General]
# Context

This issue serves as a place to start collecting resources connected to **Green Software Design**. This is part of a new initiative looking at green software from the Design, Research, Strategy & Product perspective.

Eventually these resources will be shared in a structured format.

# How to contribute

To enable everyone to contribute you have 3 options

1. Simply add a comment on this issue
2. Add entries to this [Google Docs spreadsheet](https://docs.google.com/spreadsheets/d/1sCVKIQgyCNDgDu_wcp9EPTkFHg3gUQbPGX8dcqux0JQ/edit#gid=0)
3. Add links to this [Miro board](https://miro.com/app/board/uXjVNmP33kM=/?share_link_id=927765224257)

cc: @pdematos12

**Comment by pdematos12** [comment]
Thanks @russelltrow ! I have added to the Google Spreadsheet. I also added a Type as "Guidelines & Toolkits" to distinguish from the the more generic articles and also "Community".

**Comment by russelltrow** [comment]
Some interesting points in this post: https://github.com/orgs/Green-Software-Foundation/discussions/138

I've added https://greenframe.io/ to our resource list

**Comment by olegoaer** [comment]
Hi @russelltrow 

When it comes to programming a **mobile app** (Android/iOS), a catalog of green code smells (aka green patterns) under CC license has been yielded: [https://github.com/cnumr/best-practices-mobile](https://github.com/cnumr/best-practices-mobile)

The icing on the cake: an automated code review solution comes with it, as a proof of concept.
__
Olivier, GSF [author](https://greensoftware.foundation/articles/green-coding-is-a-matter-of-code-quality) and [champion](https://champions.greensoftware.foundation/champions/olivier-le-goaer/)

**Comment by russelltrow** [comment]
@pdematos12 this very much connects to what we discussed this week.

[What Can Digital Sustainability Learn From Accessibility?](https://foundation.us5.list-manage.com/track/click?u=ddc99c7db248c3df0ef4f7d24&id=f17ae21587&e=725897333b): How we can apply lessons and patterns from accessibility to the digital sustainability movement.

**Comment by pdematos12** [comment]
> @pdematos12 this very much connects to what we discussed this week.
> 
> [What Can Digital Sustainability Learn From Accessibility?](https://foundation.us5.list-manage.com/track/click?u=ddc99c7db248c3df0ef4f7d24&id=f17ae21587&e=725897333b): How we can apply lessons and patterns from accessibility to the digital sustainability movement.

Yes very much aligned to our discussion on Tuesday.
